rtf

```diff
@@ -329,6 +331,14 @@ class Vagrant(Host):
   def start(self):
     """ Setup and run this host """
     proxy = have_proxy()
+    while "making sure that no vagrant up are running":
+      (returncode, output) = shell("pgrep -f \"vagrant up\"", fail_exit=False)
+      if returncode == 0:
+        idle = random.randint(10, 30)
```

Comment:
It seems a lot, what is the average vagrant up duration ?
This should be less

Reply:
I would say something around 2 minutes, 1 to boot the machine, another one to run the provisionner (which often installs Rudder).